openshift/manage

```diff
@@ -244,6 +246,8 @@ deployApp(){
     if dcExists ${_appName}; then
       OPERATION=update genDepls.sh -u -l
     else
+      echoWarning "\nApplying required network security policies."
```

Comment:
https://github.com/BCDevOps/openshift-developer-tools/pull/129, adds support for defining discrete NSP configurations for build and deployment environments.  NSP configuration are managed and processed just like regular build and deployment configurations which will eliminate the separate processing needed in the `manage` script.